### DISCUSSION TITLE
Important db

### DISCUSSION THREAD

**Discussion by rinrinx2** [m4mallu/clonebot#105] [Ideas]
Hi there is no need to use machine db to create database on heroku or vps anywhere

https://elephantsql.com/

exp 

DATABASE_URL = postgres://XXXXXipgi:HCYsddsz_PmRYGajzS-vQmPs0ZYJn7wO4S@isilo.db.elephantsql.com/ooyripgi

![Screenshot_1](https://user-images.githubusercontent.com/58888200/196141907-3a04388d-f4a5-4ef3-ad74-adcb1c9ae54a.png)

**Comment by rinrinx2** [comment]
![Screenshot_2](https://user-images.githubusercontent.com/58888200/196142210-a3a2327e-642a-4331-b4ba-8876bbf45b40.png)

elephantsql db good work :)

**Comment by m4mallu** [comment]
For Heroku, there is no need to create it additionally. For VPS .. this may be an alternative. Good ..

**Comment by rinrinx2** [comment]
yes sir ı add readme this alternative
